stories/Card.stories.mdx

```diff
@@ -0,0 +1,104 @@
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs';
+import { linkTo } from '@storybook/addon-links';
+import { Card, Header, Footer, Text } from '../src'; // components you used in the story
+
+
+<Meta title="Layout/Card" component={Card} />
+
+# Card
+
+The card will be used as a general interface to provide general structure to a UI component.
+It comes with very basic styling, but has a ton of customization. It includes options for headers and footers.
+The card component is made in such a way that it can be used as a container for other components.
+
+## Functionality
+
+Functionally these cards are very similar to the HTML `<div>` element. 
+The main purpose of a card is to provide information about a single piece of information.
+
+<Canvas>
+	<Story name="Standard Card" args={{ children: "This is card component" }}>
+		{(args) => <Card {...args} />}
+	</Story>
+</Canvas>
+
+## Setup
+
+Cards are setup in a way that they can be used as a container for other components. Think of it as a
+blank canvas with posibilities to add almost anything you like. So the way to use a card is to wrap
+it around any component that you want to display inside the card. Generally cards comes with a `header`
```

Comment:
The `Card` component doesn't come with `Header` and `Footer`, it implements them. Also the `Card` doesn't have a body, this might be confusing to those who read it.

Be sure to say that they can *choose* to have use those components.